### ISSUE TITLE
Add a note in the docs on how to get the database version

### ISSUE THREAD

**Issue by kfitzpatrick** [influxdb/influxdb.com#142]
See @jvshahid's comment in  https://github.com/influxdb/influxdb/issues/1072

> You actually get the version with every http response. There's a header included X-Influxdb-Version that includes the version of InfluxDB. You can probably wrap a simple function around the /ping endpoint to get back the version.

**Comment by beckettsean** [comment]
```
curl -i 'http://localhost:8086/ping'
HTTP/1.1 204 No Content
Request-Id: 70353a73-4c52-11e5-8015-000000000000
X-Influxdb-Version: 0.9.2
Date: Thu, 27 Aug 2015 00:27:45 GMT
```

**Comment by beckettsean** [comment]
Should go on the write_endpoints page.

**Comment by otoolep** [comment]
One can also simply execute:

```
service influxdb version
```

or 

```
influxd version
```

**Comment by otoolep** [comment]
This information will also be returned by `SHOW DIAGNOSTICS` when it is completed.

**Comment by beckettsean** [comment]
@rkuchan a quick one to add to the FAQ, maybe under the admin section: 

How do I find the version? (curl ping endpoint, launch CLI, check top of logs.)

**Comment by beckettsean** [comment]
closed by #283
